routes/resources.js

```diff
@@ -106,7 +107,9 @@ module.exports = function (app) {
     }
 
     return handler(params, { baseUrl: app.baseUrl }, req)
-      .then((responseBody) => respond(res, responseBody, params))
+      .then((responseBody) => {
+        respond(res, responseBody, params)
```

Comment:
Can we add a `return` or make it a one-line function to get the implicit return. I'm not sure it's necessary, but let's not change it if we don't have to

Reply:
Seems not yet changed?